topics/pubsub.md

```diff
@@ -27,8 +27,8 @@ The replies to subscription and unsubscription operations are sent in
 the form of messages, so that the client can just read a coherent
 stream of messages where the first element indicates the type of
 message. The commands that are allowed in the context of a subscribed
-client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`,
-`PING` and `QUIT`.
+client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`,
+`UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.
```

Comment:
Please see the README of this repo regarding line breaks. There is a style guide to keep diffs small to make changes easier to review.